app/controllers/api/v1/visualization/clusters_controller.rb

```diff
@@ -125,6 +155,13 @@ def self.get_cluster_viz_data(study, cluster, url_params)
           subsample = get_selected_subsample_threshold(url_params[:subsample], cluster)
           consensus = url_params[:consensus].blank? ? nil : url_params[:consensus]
 
+          default_data_fields = ['coordinates', 'expression', 'annotation', 'cells']
```

Comment:
This seems like it should be a constant for the controller as I expect it shouldn't change: `DEFAULT_DATA_FIELDS`